db/migrate/20200318135632_rename_report_templates.rb

```diff
@@ -0,0 +1,31 @@
+class RenameReportTemplates < ActiveRecord::Migration[5.2]
+  TEMPLATE_NAMES = {
+    "Ansible Inventory" => "Ansible - Ansible Inventory",
+    "Applicable errata" => "Host - Applicable Errata",
+    "Applied Errata" => "Host - Applied Errata",
+    "Entitlements" => "Subscription - Entitlement Report",
+    "Host statuses" => "Host - Statuses",
+    "Registered hosts" => "Host - Registered Content Hosts",
+    "Registered users" => "User - Registered Users",
+    "Subscriptions" => "Subscription - General Report",
+  }
+
+  class FakeReportTemplate < ApplicationRecord
+    self.table_name = 'templates'
+  end
+
+  def up
+    TEMPLATE_NAMES.each do |from, to|
+      token = SecureRandom.base64(5)
+      FakeReportTemplate.find_by(name: from)&.(:update_attributes, :name => "#{from} Backup #{token}")
+      FakeReportTemplate.find_by(name: from)&.(:update_attributes, :name => to)
```

Comment:
As the model class is simple, it won't run any orchestrations or callbacks.
```suggestion
      FakeReportTemplate.find_by(name: from)&.update(:name => "#{from} Backup #{token}")
      FakeReportTemplate.find_by(name: from)&.update(:name => to)
```

Reply:
IF you insist that it is so big issue to run Rails validations (what I don't agree with):
```suggestion
      FakeReportTemplate.find_by(name: from)&.update_columns(:name => "#{from} Backup #{token}")
      FakeReportTemplate.find_by(name: from)&.update_columns(:name => to)
```